src/transaction_values.ts

```
@@ -828,12 +868,10 @@ const parseSeaportLog = async (
     const offer = seaResult.args.offer[0];
 
     // validate token
-    if (
-      !(
-        equalIgnoreCase(offer.token, t.contract) &&
-        Number(offer.identifier) === Number(t.token_id)
-      )
-    ) {
+    if (!(
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)